tests/test_ipython_magics.py

```diff
@@ -45,12 +45,14 @@ def _validate_magics_with_black(before: Sequence[str], after: Sequence[str]) ->
     diff = difflib.unified_diff(before, after)
     result = "".join(i for i in diff if any([i.startswith("+ "), i.startswith("- ")]))
     expected = (
+        '-    "\\n",\n'
         '-    "def compute(operand1,operand2, bin_op):\\n",\n'
         '+    "def compute(operand1, operand2, bin_op):\\n",\n'
         '-    "compute(5,1, operator.add)"\n'
         '+    "compute(5, 1, operator.add)"\n'
         '-    "    ?str.splitlines"\n'
         '+    "str.splitlines?"\n'
+        '-    "\\n",\n'
         '-    "   %time randint(5,10)"\n'
```

Comment:
this one comes from
```
     "flake8_version = !pip list 2>&1 | grep flake8\n",
-    "\n",
     "if flake8_version:\n",
     "    my_var = flake8_version"
```
which...might not be a good idea

prob. best to figure out how to preserve newlines

can probably count new lines before applying autopep8, count them after, and then use that in the substitution?

should be good. might need to expand notebookinfo even further